sound/soc/sof/amd/acp.h

```diff
@@ -221,7 +224,7 @@ int acp_pcm_hw_params(struct snd_sof_dev *sdev, struct snd_pcm_substream *substr
 		      struct snd_sof_platform_stream_params *platform_params);
 
 extern struct snd_sof_dsp_ops sof_renoir_ops;
-
+extern struct snd_sof_dsp_ops sof_rembrandt_ops;
```

Comment:
might be god to keep the newline after this line

Reply:
Done